src/main/java/io/quarkus/search/app/indexing/IndexingService.java

```diff
@@ -86,6 +87,23 @@ void indexOnStartup(@Observes StartupEvent ev) {
                         t -> Log.errorf(t, "Reindexing on startup failed: %s", t.getMessage()));
     }
 
+    @Scheduled(cron = "{indexing.scheduled.cron}")
+    void indexOnTime() {
+        if (!indexingConfig.scheduled().enabled()) {
+            Log.debug("Scheduled reindex is disabled...");
+            return;
+        }
```

Comment:
This seems unnecessary, as setting `indexing.scheduled.cron=off` would also disable scheduled indexing. See javadoc of `io.quarkus.scheduler.Scheduled#cron`.